### DISCUSSION TITLE
Welcome to mac-os-opencore-asus-k501lx_lb Discussions!

### DISCUSSION THREAD

**Discussion by IFierta** [IFierta/mac-os-opencore-asus-k501lx_lb#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MaximeH38** [comment]
Hello, i have an ASUS K501lx with i5 5200u, NVIDIA GTX950M (disabled) HD5500 and Atheros AR9862 with BT AR3012
With your EFI file and some modifications my wifi works but my sleep doesn't work, and my hackintosh freeze randomly..
Can you help me?
I don't have K501LX with backlight keyboard.
![tempImagehNIxqW](https://user-images.githubusercontent.com/74606130/102620351-2f2a2100-413e-11eb-996e-262dcdf5ebfe.gif)

When he freeze, I can't open applications, but my system is responsive and in my network settings I don't have any peripheral..
Safari doesn't open.
![tempImagebIyhS0](https://user-images.githubusercontent.com/74606130/102620574-7d3f2480-413e-11eb-9ee7-9f8ca14dfd88.gif)


(sorry for my bad English, I'm French)
Thank you for your work!!!!!


[kernel.log.stream.txt](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5715811/kernel.log.stream.txt)
[boot_log.txt](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5715839/boot_log.txt)
[system_log.txt](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5715840/system_log.txt)

**Reply by IFierta** [reply]
Are you using ATH9KFixup?
If so, replace your wi-fi module with a Broadcom BCM94352HMB. The system may freeze due to ATH9KFixup kext.

**Reply by MaximeH38** [reply]
Oh! Thank you for reply, your wifi module BCM94352HMB works on Catalina or Big Sur?
I didn't think you answer me, thank you so much!

**Reply by IFierta** [reply]
> Oh! Thank you for reply, your wifi module BCM94352HMB works on Catalina or Big Sur?
> I didn't think you answer me, thank you so much!

I didn't answer for a long time because I didn't see your message.
Yes my module is BCM94352HMB, works fine in Catalina and Big Sur.
Is your sleep working? Have you fixed it?

**Reply by IFierta** [reply]
![Item-0_и_Bluetooth_и_WiFi](https://user-images.githubusercontent.com/26610620/104518167-20d60480-5619-11eb-8569-8825b82f1c83.jpg)

**Reply by MaximeH38** [reply]
No it's not working, sometimes work but blackscreen at wake...
With your EFI file my system won't boot on big sur or catalina install, stuck at apple logo. When Catalina was installed on my laptop it wasn't all your files.
I have put "-v" on bootargs in my config.plist look the picture
EDIT : This is why my installer won't boot.

![unnamed (1)](https://user-images.githubusercontent.com/74606130/104555522-53debf00-563e-11eb-9563-347df866f597.jpg)

After enable AppleCpuPmCfgLock and AppleXcpmCfgLock mac os boot but obtain kernel panic
![unnamed (2)](https://user-images.githubusercontent.com/74606130/104559340-87bce300-5644-11eb-8105-783a8523f2dc.jpg)

**Reply by IFierta** [reply]
In the second screenshot you have an error in ACPI.

Try it with this EFI, do not change anything in it, do not add ATH9KFixup kext. Unlock the CFG Lock register (MSR E2). There is an unlock utility in the EFI folder, it is already enabled, you only need to boot from this EFI and select CFG Lock. Let me know the result.

[EFI.zip](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5817410/EFI.zip)

**Reply by IFierta** [reply]
>>No it's not working, sometimes work but blackscreen at wake...

Read this issue to the end ( [Display sleep](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/issues/7)). Pay attention to my last post. ( https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/issues/7#issuecomment-745954497 )

**Reply by MaximeH38** [reply]
Finally, with your « base » EFI folder in your repository, my laptop start on big sur very well!
Wake doesn’t work. With the EFI folder above, my laptop start well too.
Wake doesn’t work because my screen was not recognized ? Brightness work, sound work, ethernet work, HD Graphics work, FN keys work, touchpad work, all usb ports work. 
Oh! When i start in verbose i saw : disks2s5s1 : device is not readable.
![4AFBB903-E41C-4852-9D5D-6C31EA93AC20](https://user-images.githubusercontent.com/74606130/104695504-2876d580-570d-11eb-8b3f-36336cdeccb4.jpeg)
![138D31D1-A58E-49F2-BF9D-009D0E80CE64](https://user-images.githubusercontent.com/74606130/104695576-46443a80-570d-11eb-8bd1-2203cca6d43c.jpeg)

On this post wifi work on big sur 
https://www.reddit.com/r/hackintosh/comments/ktdjck/success_acer_xc605_oc_065_big_sur_111_everything/

![4E75846B-5EB0-4F85-9D71-1B38A1E4E360](https://user-images.githubusercontent.com/74606130/104695995-d97d7000-570d-11eb-9623-3350f62580f6.png)

Can you help me for this my friend ?
Thank you so much for your help!

**Reply by IFierta** [reply]
Replace your config with this one. See if wi-fi works or not.

[config.zip](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5819627/config.zip)

>>Wake doesn’t work because my screen was not recognized ? 

Yes. The screen does not work on all asus k501lx-lb. Something was changed in the video driver, starting from sierra 10.13.4  the screen wakes up to a black screen. In big sur, the screen works better than before, but not well enough.

>>Oh! When i start in verbose i saw : disks2s5s1 : device is not readable.

Do you still have this problem?

**Reply by MaximeH38** [reply]
Oh god! The wifi works well! 
![Capture d’écran 2021-01-15 à 10 48 44](https://user-images.githubusercontent.com/74606130/104709747-91674900-571f-11eb-8125-dd5d2b759a9e.png)
![Capture d’écran 2021-01-15 à 10 50 43](https://user-images.githubusercontent.com/74606130/104709775-988e5700-571f-11eb-81ed-49615be0e549.png)

Yes! device is not readable. but I have no problem. so strange!

If wake doesn't work is not a serious problem.

**Reply by IFierta** [reply]
> If wake doesn't work is not a serious problem.

The awakening of the screen in big sur is fine, only it does not wake up the first time. Configure screen wake up as written here: https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/issues/7#issuecomment-743965739

If the screen wakes up to a black screen, use a key combination to put the screen to sleep and wake it up again, do this until the screen wakes up.

**Reply by IFierta** [reply]
These are the color profiles for your screen.
Copy color profiles in /Library/ColorSync/Profiles/Displays
[ColorProfiles.zip](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5819834/ColorProfiles.zip)

**Reply by IFierta** [reply]
Do not change smbios to MacBookPro, use MacBookAir, with this smbios laptop works better.

**Reply by MaximeH38** [reply]
With bettertouchtool the display go to sleep but after that the display is black... (with backlight on)

It’s okay, my smbios is MacbookAir7,2 

Thank you for color profile! Much better Colors.
My wifi cause freeze, I have download an another AirportAtheros40 kext for big sur and no freeze after 1hour....

**Reply by IFierta** [reply]
>>With bettertouchtool the display go to sleep but after that the display is black... (with backlight on)

Try to sleep and wake up the screen several times until the desktop appears. 

Open your console, find displaypolicyd.0 / 2 / 0.log.   Look, are there lines like this?

 ERROR: port 2 kAGDCPortStatus returns Error, skipping ...
NOTICE: accepted No_Media_Mode enter, 
 LinkUpdate: 1.0 event: Change [2]
 NOTICE: accepted No_Media_Mode exit, 
NOTICE: Addr: 1.0 event: Change [2] source: 0

**Reply by MaximeH38** [reply]
Hello friend!
Okay, after a couple of hours my hackintosh freeze but not completly (only safari and network settings froze...) 

look my log file
[displaypolicyd.0:2:0.log](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5824124/displaypolicyd.0.2.0.log)

With BTT my screen wake up!!!

**Reply by IFierta** [reply]
> With BTT my screen wake up!!!

Hello friend. Fine. Now you know how to wake up your screen.   :)

>>Okay, after a couple of hours my hackintosh freeze but not completly (only safari and network settings froze...)

Replace your Wi-Fi card with a Broadcom or buy a compatible USB Wi-Fi. Otherwise, your problem with network freezing and safari will remain.

I hope I helped you solve all the problems?  :)

**Reply by MaximeH38** [reply]
Fine, broadcom wifi cards compatible with combo Bluetooth? (And AirDrop?)

do you know how to repair imessage?
i never managed to make it work :)
Thank you!

PS : i have updated my big sur version 11.0.1 beta to 11.1 with no problems! Run so smooth!

**Reply by IFierta** [reply]
I have a BCM94352HMB and an airdrop works in it, I can't say anything about other non-native cards.
how to make imessage work, I'll write about it later.

**Reply by MaximeH38** [reply]
Your Bluetooth combo works on your BCM94352HMB?

My HDMI port doesn’t work, you have a solution? :)

okay, thank you!!

**Reply by MaximeH38** [reply]
iMessage is so hard on an hackintosh.. 😩😩

**Reply by IFierta** [reply]
>>My HDMI port doesn’t work, you have a solution? :)

I haven't tested HDMI, I don't use it. Read my HDMI.md file. Maybe I'll try to make HDMI next weekend, but I can't promise anything.

**Reply by MaximeH38** [reply]
Oh my god, iMessage works !!
Sleep works without BTT, my screen wake by itself, its so strange???
![Capture d’écran 2021-01-17 à 16 17 43](https://user-images.githubusercontent.com/74606130/104847369-d89c3800-58df-11eb-9af7-b765efd0b3c3.png)

look my solution for black screen after wake without BTT (you have to enable password after wake (immediatly))

https://youtu.be/6C2fjfpCWCk

**Reply by IFierta** [reply]
> Sleep works without BTT, my screen wake by itself, its so strange???
> ![Capture d’écran 2021-01-17 à 16 17 43](https://user-images.githubusercontent.com/74606130/104847369-d89c3800-58df-11eb-9af7-b765efd0b3c3.png)
> 
> look my solution for black screen after wake without BTT (you have to enable password after wake (immediatly))
> 
> https://youtu.be/6C2fjfpCWCk

Good morning! :)
Solution without BTT, good. You are well done. 👍 
There is nothing strange in this, without BTT the screen can wake up, the main thing is to put it into sleep mode and wake it up again. In your video, you do exactly that, but without the help of BTT.  Maybe someday Apple will fix the video driver. This issue was caused by changes in the video driver.

Do not use OpenCore Configurator to edit the configuration, it may mess up your config.
Use https://github.com/ic005k/PlistEDPlus
It's not as convenient as OpenCore Configurator, but it won't break anything for you.
Or use this: https://github.com/ic005k/QtOpenCoreConfig

> Oh my god, iMessage works !!

The strangest thing about the iMessage is that it does not work without rom and mlb from the MacBook. I gave you rum and mlb from MacbookAir 2015
I'm glad that everything works for you. :)


I still need to fix ssdt batt, because of it the touchpad can get stuck. Are you experiencing this problem?

**Reply by MaximeH38** [reply]
Good morning my friend!
Oh okay, i use it. Opencore configurator is only for mount my EFI 😂😂
You gave me a real MacBook serial ?! Really?
A random serial generated checked on Apple Coverage when it say « invalid serial number » doesn’t work? Some tutorial say : use my MAC adress of Ethernet card and paste it on « ROM » section.
When you give me a serial for IMessage i have to deconnect?
Yes, my touchpad freeze suddenly for 1second. That does not bother

**Reply by IFierta** [reply]
I generated a good serial number for you, do not change it.
Yes. Better to use mac ethernet address. But that doesn't always work. It does not work for me. Works only with rom and mlb from real Mac. Check it out, generate a new mlb and add mac ethernet address instead of rom. See if imessage will work. If it doesn't work, write to me. Yes, it is better to turn off imessage, but it is not necessary.  Replace your ssdt with this one. Check if the touchpad will stuck. 
[SSDT-BATT-ASUS.aml.zip](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5828345/SSDT-BATT-ASUS.aml.zip)

**Reply by IFierta** [reply]
>Oh okay, i use it. Opencore configurator is only for mount my EFI 😂😂

try it use: https://github.com/Andrej-Antipov/MountEFI/releases

**Reply by MaximeH38** [reply]
How generate a new mlb?
I have send a mail at your e-mail address for serial.
My serial is correct? When you gave me the config.plist ???
Oh! Okay, i have a mail from you!
> > Oh okay, i use it. Opencore configurator is only for mount my EFI 😂😂
> 
> try it use: https://github.com/Andrej-Antipov/MountEFI/releases

Okay i use it! :)

**Reply by IFierta** [reply]
> How generate a new mlb?
> I have send a mail at your e-mail address for serial.
> My serial is correct? When you gave me the config.plist ???
> Oh! Okay, i have a mail from you!
> 
> > > Oh okay, i use it. Opencore configurator is only for mount my EFI 😂😂
> > 
> > 
> > try it use: https://github.com/Andrej-Antipov/MountEFI/releases
> 
> Okay i use it! :)

Yes, I gave you a good serial number.
New mlb: C02543108CDG91Q1H.

**Reply by MaximeH38** [reply]
> C02543108CDG91Q1H

it is better to use the one you sent me by email?

it's a fail.... iMessage kick after log in 
![Capture d’écran 2021-01-18 à 08 27 34](https://user-images.githubusercontent.com/74606130/104884731-2314c780-5967-11eb-9b68-8e2ef98384a4.png)

Resolved ! Oops, sorry 🤣

**Reply by IFierta** [reply]
> > C02543108CDG91Q1H
> 
> it is better to use the one you sent me by email?
> 
> it's a fail.... iMessage kick after log in
> ![Capture d’écran 2021-01-18 à 08 27 34](https://user-images.githubusercontent.com/74606130/104884731-2314c780-5967-11eb-9b68-8e2ef98384a4.png)
> 
> Resolved ! Oops, sorry 🤣

imessage works with mac ethernet?

**Reply by MaximeH38** [reply]
> > > C02543108CDG91Q1H
> > 
> > 
> > it is better to use the one you sent me by email?
> > it's a fail.... iMessage kick after log in
> > ![Capture d’écran 2021-01-18 à 08 27 34](https://user-images.githubusercontent.com/74606130/104884731-2314c780-5967-11eb-9b68-8e2ef98384a4.png)
> > Resolved ! Oops, sorry 🤣
> 
> imessage works with mac ethernet?

Oops! I don't saw your reply! 
Nope, with my eth0 Mac address iMessage doesn't work...
With the SSDT-BATT-ASUS.aml (2hours ago) my touchpad freeze again.

**Reply by MaximeH38** [reply]
[EFI.zip](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5842871/EFI.zip)
Don't panic I have removed wifi kexts because it's unstable. I use USB Tethering with my iPhone!

**Reply by IFierta** [reply]
Wi-Fi has nothing to do with it, I'm trying to understand why the touchpad does not work after sleep.

**Reply by MaximeH38** [reply]
> Wi-Fi has nothing to do with it, I'm trying to understand why the touchpad does not work after sleep.

Oh okay ^^ When touchpad stopped working after sleep my wifi kext was still injected. maybe they were causing the problem?
I was trying to edit the kext to make it more stable (remove other IOKitPersonnalities) and edit IOProduct and IOVendor.
I ended up giving up and waiting for the card I ordered 🤣

**Reply by IFierta** [reply]
> > Wi-Fi has nothing to do with it, I'm trying to understand why the touchpad does not work after sleep.
> 
> Oh okay ^^ When touchpad stopped working after sleep my wifi kext was still injected. maybe they were causing the problem?
> I was trying to edit the kext to make it more stable (remove other IOKitPersonnalities) and edit IOProduct and IOVendor.
> I ended up giving up and waiting for the card I ordered 🤣

If you are not using wi-fi, remove these kexts.

**Reply by MaximeH38** [reply]
> > > Wi-Fi has nothing to do with it, I'm trying to understand why the touchpad does not work after sleep.
> > 
> > 
> > Oh okay ^^ When touchpad stopped working after sleep my wifi kext was still injected. maybe they were causing the problem?
> > I was trying to edit the kext to make it more stable (remove other IOKitPersonnalities) and edit IOProduct and IOVendor.
> > I ended up giving up and waiting for the card I ordered 🤣
> 
> If you are not using wi-fi, remove these kexts.

It's impossible to use wifi, system partially freeze 5/20minutes after wifi is connected. 
I only use USB Tethering.

**Comment by wenwei7532** [comment]
Thank you brother, through your way, now WiFi sleep can work.

**Reply by IFierta** [reply]
Fine. I'm glad that everything works for you, brother.

**Comment by wenwei7532** [comment]
Using the latest 1.51 efi, the trackpad is a bit stuck and not smooth

**Reply by IFierta** [reply]
Disable SSDT-BATT-ASUS.aml, see if trackpad gets stuck or not.

**Reply by wenwei7532** [reply]
Disable SSDT-BATT-ASUS.aml, it returns to normal.

**Reply by IFierta** [reply]
I will try to fix it.

**Reply by IFierta** [reply]
Try this ssdt. Let me know if the touchpad with this ssdt stuck.
[SSDT-BATT-ASUS.aml.zip](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5828810/SSDT-BATT-ASUS.aml.zip)

**Comment by IFierta** [comment]
[VoodooPS2Controller-2.2.0-DEBUG.zip](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/5842560/VoodooPS2Controller-2.2.0-DEBUG.zip)

**Reply by MaximeH38** [reply]
It's works!!

**Comment by dtadptvl** [comment]
**Hi IFierta. Thank you for your contribution. It's very lucky of me to find your config here. However, it comes to some issues when I try booting using your EFI:**

1. When I use your config to boot into a fully-installed partition which was set up from Clover, I encounter a black screen with a forbidden sign for 15 seconds then the computer gets rebooted.
![image](https://user-images.githubusercontent.com/51063200/110969788-abbe4b80-838b-11eb-9fc7-09f24f6aa3a8.png)

2. When I boot into a fresh installer drive, it stucks at apple logo for 5 seconds then reboots over and over again.

**This is the system I'm using:**
`Mojave 10.14.3`
`Asus K501LB`
`Intel i5-5200U with HD Graphics 5500`
`RAM 8GB Dual`
`SSD Intel 540s`
`Geforce 940M (should be disabled I know)`

**Kexts**
![image](https://user-images.githubusercontent.com/51063200/110969113-e5db1d80-838a-11eb-93db-6b5f5b1f6533.png)

Please send help. Thank you very much.

**Reply by MaximeH38** [reply]
Hello, I'm not iFierta but I can help you.
In first, have you disabled CFG Lock? You can disable it from Opencore entry.

Your Realtek Ethernet Kext is not correct. Open your folder and copy the .kext on the Kext folder inside your OC folder.
![Capture d’écran 2021-03-12 à 18 49 49](https://user-images.githubusercontent.com/74606130/110978636-edd39700-8363-11eb-87b8-6e343204b679.png)

For disable CFGLock you have to boot into OpenCore and press the spacebar ;)
(Check if CFGLock.efi is on your Tool folder on your OpenCore EFI Folder.)

If the CFGLock is not disabled your ASUS K501LX your system won't boot.

![Capture d’écran 2021-03-12 à 18 48 29](https://user-images.githubusercontent.com/74606130/110978356-946b6800-8363-11eb-9f4d-328f0c0b34df.png)

**Reply by dtadptvl** [reply]
@MaximeH38 I’ve followed your instructions but only Reset NVRAM has shown up. But thanks for your suggestions, after I enable CFG Lock by enabling **AppleCpuPmCfgLock** and **AppleXcpmCfgLock** in the config.plist, it succeeds in booting into the OS. You’re truly the savior.
But one more issue emerges, waking the computer from hibernation causes black screen. Then the SSD and HDD are at not-working state as the disk light doesn’t flash. Could you please help me with this? Thank you

**Reply by MaximeH38** [reply]
Send me your EFI file 😉
https://file.re/upload-form/
I can help you. 
You have to add CFGLock in your config.plist 😉
![Capture d’écran 2021-03-13 à 11 55 52](https://user-images.githubusercontent.com/74606130/111027961-c1fff200-83f3-11eb-8759-9bc976f8d71b.png)
Hibernation or sleep?
Your screen is black but when you press the « power » button 2 Times your screen wake up or not? Press it for 1second and re-press it again. If you are in sleep mode your screen will wake up with this trick.

**Reply by dtadptvl** [reply]
@MaximeH38 It’s very kind of you. Fortunately I managed to work it out myself. I’m currently struggling with Big Sur. I don’t know why iy doesn’t boot. Could you please share your EFI? Thanks

**Reply by MaximeH38** [reply]
[EFI K501LX Big Sur.zip](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/6139586/EFI.K501LX.Big.Sur.zip)
ENABLE CFGLock on Opencore Bootpicker, don't enable AppleCpuPmCfgLock and AppleXcpmCfgLock.
You have to select CFGLock on Opencore entry, (press spacebar on OpenCore to show auxiliary) and type enter on it. CFGLock show you a value (0 or 1) if you have 1, press "Y" and shutdown the computer. reboot on Mac OS and voilà! Your CFGLock is unlocked.

Refill Serial, MLB and ROM on my EFI file. if you have wifi card put your kext.
Don't use Opencore Configurator, use Propertree (open your config.plist and click on Clean OC Snapshot, select your EFI file)
My EFI works at 100% Sleep, Hibernate, Trackpad, Battery, Ethernet, Sound, Headphones... (you have to install ALCPlugFix for headphones distortions) 
iFierta help me and my EFI is 100% working on Big Sur! 
With my wifi card I have AirDrop, Continuity, Handoff
All of iServices works! iMessage, FaceTime......
For this you have to get a good ROM and MLB and a valid serial unused.

**Reply by dtadptvl** [reply]
![4FAF7F0B-A45E-4E71-9B8D-A2BC4D400BB8](https://user-images.githubusercontent.com/51063200/111122762-0ed4fb80-85a1-11eb-9904-d8170d478501.jpeg)

I got the this error. NVRAM has been cleaned. Booting from my EFI causes the same issue. I don’t have an USB stick with me so I flash the installer to a local partition using R-Drive. Is it the cause?

**Reply by MaximeH38** [reply]
It could be the cause, i've never done that 😅
Can you replace with this config.plist? 

[config.plist.zip](https://github.com/IFierta/mac-os-opencore-asus-k501lx_lb/files/6139766/config.plist.zip)
Show me the kernel panic I have enabled verbose.

**Reply by dtadptvl** [reply]
It stops here
![B7D6A957-145E-4681-A012-0A30403896A4](https://user-images.githubusercontent.com/51063200/111124529-30cf7d80-85a3-11eb-9c07-99d2e931f116.jpeg)
Then turns to this 
![8538D020-2FAA-4A27-84C2-6CE4939EE8CF](https://user-images.githubusercontent.com/51063200/111124588-404ec680-85a3-11eb-9290-da1d920d7312.jpeg)

**Reply by MaximeH38** [reply]
It really looks like it comes from the fact that you didn't use a USB key, I only see that... 
I can read "Waiting for root devices" The system can't access system files for booting..

**Reply by dtadptvl** [reply]
Yeah I guess so.
When I flash Mojave installer to a local partition with Acronis True Image, it works well.
But now I can only find Big Sur installer with R-Drive... Guess I have to grab a USB stick to get things done
Thank you very much @MaximeH38. Really appreciate your help

**Reply by MaximeH38** [reply]
Find a USB key and tell me if it works. You have to create USB key from MacOS 
(i have used a Virtual Machine when I had no hackintosh)
It's not advisable but it works, with VMWare workstation and the fix for install MacOS, after that you have to select the BigSur ISO, and boot on it. with your virtual machine you open the iso file on Mac OS, grab the "Install MacOS Big Sur.app" and drag to Applications folder. after that you have to create your USB key with the terminal 😉
A lot of work when you don't have a Mac....

I don't know the software R-Drive or Acorns True Image.. 

Look at how CFGLock works 
https://youtu.be/H_hhOMotfB0

**Reply by dtadptvl** [reply]
I have tried using Install Disk Creator on Mac to write the installer to the partition but somehow it forbidded me to erase.
The reason why I prefer a local hard disk over a USB is that I once kill 2 USBs due to creating macOS installer lol.
Now I guess I have to grab another USB and hope it stays well 😂

**Reply by MaximeH38** [reply]
It's not normal, I've never killed a USB key, you just have to repartition it.
Open Disk Utility on Mac OS
click on "presentation" select Show devices"
Select your disk on the left section.
Click Erase and name your disk "Install"7
Choose Mac OS Extended (Journaled) and Partition Table GUID.
![Capture d’écran 2021-03-15 à 11 12 52](https://user-images.githubusercontent.com/74606130/111138565-5d25d280-8580-11eb-8bea-af6c702b4105.png)

After that, open terminal, type "sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --volume /Volumes/Install"

Confirm with "Y"
After that, open your EFI partition with MountEFI 
https://github.com/Andrej-Antipov/MountEFI
(On MacOS)

And drop my EFI folder on it. Not OC and BOOT!!!!
the whole EFI Folder.
example: EFI Volume -> EFI folder at root -> OC and BOOT in EFI folder.


PS : You can't use an APFS drive to make a bootable installation of Mac OS, this is why your installation fail with Install Disk Creator.
Don't use it, use terminal ;)

Format your drive in Mac OS Extended Journaled (with GUID Partition Table)
It's important.

**Reply by dtadptvl** [reply]
Eventually I have decided to do OTA Update. Although it took an eternity to finish, the result is perfect. My laptop now works smoothly. Thank you for your dedication Maxime.
![image](https://user-images.githubusercontent.com/51063200/111154700-19ee5280-85c6-11eb-901f-c6c6a7c30132.png)

**Reply by MaximeH38** [reply]
Good! Hide your serial number ;)!!!!

**Reply by dtadptvl** [reply]
Very kind of you but don't worry. It can be easily generated with this tool
https://github.com/corpnewt/GenSMBIOS

**Reply by MaximeH38** [reply]
Your system run smoothly? More than Mojave or the same?
Startup is longer than Catalina I think

**Reply by dtadptvl** [reply]
I feel quite the same but technically Activity Monitor shows the CPU and Memory works a bit harder, about 10% more than they do in Mojave

**Reply by MaximeH38** [reply]
Okay, good!

**Reply by dtadptvl** [reply]
Just one more thing, sleep works perfectly when CSM in BIOS is enabled

**Reply by MaximeH38** [reply]
Yes!!😁

**Comment by dtadptvl** [comment]
I install ALCPlugFix (version of goodwin) the terminal responses this
![image](https://user-images.githubusercontent.com/51063200/111589105-01b14a00-87f7-11eb-93b4-a73f29abace9.png)
Please send help 🥺

**Reply by MaximeH38** [reply]
Type « cd on your terminal and drag the ALCPLUGFIXSWIFT folder and type enter. 
after that, drag install.sh on your terminal. And after drag sample.plist when alcplugfix ask you to put the .plist 😉

**Reply by dtadptvl** [reply]
Awesome! My audio jack works perfectly. You save me again, Maxime. Looks like you're always on the internet lol

**Reply by MaximeH38** [reply]
I'm at home, when I get an email and I'm available I come to help😂😁

**Reply by dtadptvl** [reply]
Have a nice day then!

**Reply by MaximeH38** [reply]
You too, if you have a problem i can help you. (Also iFierta)
